src/app/scripts/cac/control/cac-control-itinerary-list.js

```diff
@@ -106,10 +110,30 @@ CAC.Control.ItineraryList = (function ($, Handlebars, Utils) {
         events.trigger(eventNames.itineraryClicked, itinerary);
     }
 
+    /**
+     * Handle hover event on itinerary list item
+     */
+    function onItineraryHover() {
+        var itineraryId = this.getAttribute('data-itinerary');
```

Comment:
Alternatively, this can be done with: `$(this).data('itinerary')`, which is arguably slightly cleaner.